### PR TITLE
Extract BigQuery Data Transfer

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: google-api-grpc/proto-google-cloud-bigquerydatatransfer-v1/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Google Cloud Java Client for Kubernetes Engine
```
Comment (pmakani) [inline google-api-grpc/proto-google-cloud-bigquerydatatransfer-v1/README.md]: BigQuery Data Transfer

File: google-cloud-clients/google-cloud-bigquerydatatransfer/README.md [hunk 2]
```diff
@@ -1,135 +1,3 @@
-Google Cloud Java Client for Bigquery Data Transfer
-====================================
+# Google Cloud Java Client for Kubernetes Engine
```
Comment (pmakani) [inline google-cloud-clients/google-cloud-bigquerydatatransfer/README.md]: BigQuery Data Transfer

Comment (chingor13) [reply google-cloud-clients/google-cloud-bigquerydatatransfer/README.md]: Fixed

File: google-api-grpc/proto-google-cloud-bigquerydatatransfer-v1/README.md (hunk 1 above)
Comment (chingor13) [reply google-api-grpc/proto-google-cloud-bigquerydatatransfer-v1/README.md]: Fixed

File: google-api-grpc/proto-google-cloud-bigquerydatatransfer-v1/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Google Cloud Java Client for BigQuery Data Transfer
+
+This client has moved to https://github.com/googleapis/java-container
```
Comment (pmakani) [inline google-api-grpc/proto-google-cloud-bigquerydatatransfer-v1/README.md]: java-bigquerystorage

**Pull request merged**